internal/rpc/tls_server.go

```diff
@@ -96,10 +88,10 @@ func (s *tlsServer) start(params *ServerParams) (func(), error) {
 	serverStartWaiter.Add(1)
 	go func() {
 		serverStartWaiter.Done()
-		tlsServerLogger.Infof("Serving gRPC: %s", s.grpcLh.AddrString())
+		serverLogger.Infof("Serving gRPC-TLS: %s", s.grpcLh.AddrString())
```

Comment:
Any reason to use different loggers for server.go (logger) and for the tls / insecure files (serverlogger) ?

Reply:
I didn't submit that change to upstream by accident. It's the way you think it should be.